troubleshooting.md

```diff
@@ -210,19 +210,13 @@ cannot find newuidmap: exec: "newuidmap": executable file not found in $PATH
 
 Install a version of shadow-utils that includes these executables.  Note RHEL7 and Centos 7 will not have support for this until RHEL7.7 is released.
 
-### 10) podman fails to run in user namespace because /etc/subuid is not properly populated.
+### 10) rootless setup user: invalid argument
 
-Rootless podman requires the user running it to have a range of UIDs listed in /etc/subuid and /etc/subgid.
+Rootless podman requires the user running it to have enough subUIDs and subGIDs for running the container.
```

Comment:
I think you still need to mention the files.  Perhaps something like
```
While podman will run with a single user mapped to root inside of the container.  If the container image requires additional uids mapped into the container.  These uids must be mapped for the user in /etc/subuid and /etc/subgid.
```

Reply:
Concur with @rhatdan, I'd suggest changing his suggestion just slightly to: 
Podman will run with a single user mapped to root inside of the container.  However, if the container image requires additional uids, they must be mapped into the container.  These uids are mapped for the user in /etc/subuid and /etc/subgid files.

Reply:
restored